server/routes/getChainEvents.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response, NextFunction } from 'express';
+import { sequelize, DB } from '../database';
+
+const getChainEvents = async (
```

Comment:
Why do we need this route?

Reply:
This is how we get the vote-cast events to calculate pagination for which members to load on delegation pages first.